setup.py

```diff
@@ -43,6 +43,6 @@
     keywords='ITK InsightToolkit',
     url=r'https://itk.org/',
     install_requires=[
-        r'itk>=5.2rc2'
+        r'itk>=5.3rc3'
```

Comment:
Maybe also update `version` on line 15.

Reply:
Yes, we will do it once everything is finalized before the conference.
We have to update the readme also.

Reply:
Are we removing Python 36, 37 version support?

Reply:
I think we are dropping 3.6, but keeping 3.7.